### DISCUSSION TITLE
Preferred shortcut keys for "Redo"

### DISCUSSION THREAD

**Discussion by Minothor** [itext/i7j-rups#17] [Polls]
As I'm expanding the Undo/Redo functionality to other areas of Rups, I find myself switching mindsets between Intelli-J's shortcuts and Rups' shortcuts.

So I wanted to ask, which default would be easiest for your own workflow?

**Comment by MatthiasValvekens** [comment]
¿Por qué no los dos?

**Reply by MyMilkedEek** [reply]
You laugh, but we could leave it up to the user through the preferences.

**Comment by michaeldemey** [comment]
I'm always in favor of using the shift key as an inverse of the original shortcut. Like alt tab vs alt shift tab.

**Comment by Minothor** [comment]
To be honest, editable in prefs was always an option, but there's got to be
a default config to change it from - hence this poll really!

**Reply by MatthiasValvekens** [reply]
As long as I get the option to change it, I don't care either way :). If I had to choose one option, I would prefer Ctrl+Y, though.

Admittedly, I don't have the muscle memory issue with IntelliJ because I use IDEAVim, so my Undo shortcut there is just `u`. So feel free to set the default to whatever makes sense to you!
